lib/rack/utils.rb

```diff
@@ -223,6 +223,21 @@ def parse_cookies_header(header)
     end
 
     def add_cookie_to_header(header, key, value)
+      warn "add_cookie_to_header is deprecated and will be removed in Rack 3.1"
```

Comment:
Need to add `uplevel: 1` to this warn call and all other warn calls added.

Reply:
Oh, thanks, damn, I didn't learn from this last time you mentioned it.